### PR TITLE
Add PL translation for #480

### PR DESCRIPTION
**Pull request by Koziolek**

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @Koziolek!!  @kdebowski could you give this a look if you're free?

File: pl/lessons/basics/basics.md [hunk 1]
```diff
@@ -98,6 +98,20 @@ iex> :true === true
...
 ```
 
+Nazwy modułów w Elixirze są atomami. Na przykład `MyApp.MyModule` jest poprawnym atomem nawet wtedy, gdy moduł o takiej nazwie nie istnieje.
+
+```elixir
+iex> is_atom(MyApp.MyModule)
+true
+```
+
+Atomy służą też jako odwołania do bibliotek Erlanga, równierz tych wbudowanych.
```
Comment (kdebowski) [inline pl/lessons/basics/basics.md]: równierz -> również

**Comment by kdebowski** [comment]
@Koziolek I've added one note.

**Comment by Koziolek** [comment]
Nice typo...

**Pull request merged**